package.json

```diff
@@ -32,9 +32,6 @@
   "engines": {
     "node": ">=16"
   },
-  "pre-commit": [
```

Comment:
Should we discuss this? It feels out of scope of the coverage report.

Reply:
Let's dicuss it, yes. I'll start a thread on Slack.